cirrus/server/cirrus/main.py

```diff
@@ -256,14 +258,32 @@ async def compute_features(
     client_feature_configuration: dict[str, Any] = (
         app.state.fml.compute_feature_configurations(enrolled_partial_configuration)
     )
-
-    await record_metrics(
-        enrolled_partial_configuration=enrolled_partial_configuration,
+    # Enrollments data
+    enrollment_data = collate_enrollment_metric_data(
+        enrolled_partial_configuration,
         client_id=request_data.client_id,
-        nimbus_preview_flag=nimbus_preview or False,
+        nimbus_preview_flag=nimbus_preview,
     )
 
-    return client_feature_configuration
+    # Record metrics
+    await record_metrics(enrollment_data)
+
+    response = {
+        "Features": client_feature_configuration,
+        "Enrollments": [
```

Comment:
why capital keys?

Reply:
no specific reason, its something we agreed upon as part of the plan